<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoordinateConversionClass</class>
 <widget class="QMainWindow" name="CoordinateConversionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CoordinateConversion</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::South</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="icon">
        <iconset resource="CoordinateConversion.qrc">
         <normaloff>:/CoordinateConversion/res/data.png</normaloff>:/CoordinateConversion/res/data.png</iconset>
       </attribute>
       <attribute name="title">
        <string>数据</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="widget1" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="icon">
        <iconset resource="CoordinateConversion.qrc">
         <normaloff>:/CoordinateConversion/res/picture.png</normaloff>:/CoordinateConversion/res/picture.png</iconset>
       </attribute>
       <attribute name="title">
        <string>图形</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="icon">
        <iconset resource="CoordinateConversion.qrc">
         <normaloff>:/CoordinateConversion/res/report.png</normaloff>:/CoordinateConversion/res/report.png</iconset>
       </attribute>
       <attribute name="title">
        <string>报告</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="widget3" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>计算</string>
    </property>
    <addaction name="BLHtoXYZ"/>
    <addaction name="XYZtoBLH"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="spotsPicture"/>
    <addaction name="enlarge"/>
    <addaction name="reduce"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="helpAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openFile"/>
   <addaction name="saveFile"/>
   <addaction name="BLHtoXYZ"/>
   <addaction name="XYZtoBLH"/>
   <addaction name="spotsPicture"/>
   <addaction name="enlarge"/>
   <addaction name="reduce"/>
   <addaction name="helpAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="openFile">
   <property name="icon">
    <iconset resource="CoordinateConversion.qrc">
     <normaloff>:/CoordinateConversion/res/openFile.png</normaloff>:/CoordinateConversion/res/openFile.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="icon">
    <iconset resource="CoordinateConversion.qrc">
     <normaloff>:/CoordinateConversion/res/saveFile.png</normaloff>:/CoordinateConversion/res/saveFile.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="BLHtoXYZ">
   <property name="icon">
    <iconset resource="CoordinateConversion.qrc">
     <normaloff>:/CoordinateConversion/res/BLHXYZ.png</normaloff>:/CoordinateConversion/res/BLHXYZ.png</iconset>
   </property>
   <property name="text">
    <string>BLH→XYZ</string>
   </property>
  </action>
  <action name="XYZtoBLH">
   <property name="icon">
    <iconset resource="CoordinateConversion.qrc">
     <normaloff>:/CoordinateConversion/res/BLHXYZ.png</normaloff>:/CoordinateConversion/res/BLHXYZ.png</iconset>
   </property>
   <property name="text">
    <string>XYZ→BLH</string>
   </property>
  </action>
  <action name="spotsPicture">
   <property name="icon">
    <iconset resource="CoordinateConversion.qrc">
     <normaloff>:/CoordinateConversion/res/spotsPicture.png</normaloff>:/CoordinateConversion/res/spotsPicture.png</iconset>
   </property>
   <property name="text">
    <string>点位图</string>
   </property>
  </action>
  <action name="report">
   <property name="icon">
    <iconset resource="CoordinateConversion.qrc">
     <normaloff>:/CoordinateConversion/res/report.png</normaloff>:/CoordinateConversion/res/report.png</iconset>
   </property>
   <property name="text">
    <string>报告</string>
   </property>
  </action>
  <action name="enlarge">
   <property name="icon">
    <iconset resource="CoordinateConversion.qrc">
     <normaloff>:/CoordinateConversion/res/enlagre.png</normaloff>:/CoordinateConversion/res/enlagre.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="reduce">
   <property name="icon">
    <iconset resource="CoordinateConversion.qrc">
     <normaloff>:/CoordinateConversion/res/reduce.png</normaloff>:/CoordinateConversion/res/reduce.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="icon">
    <iconset resource="CoordinateConversion.qrc">
     <normaloff>:/CoordinateConversion/res/help.png</normaloff>:/CoordinateConversion/res/help.png</iconset>
   </property>
   <property name="text">
    <string>查看说明</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CoordinateConversion.qrc"/>
 </resources>
 <connections/>
</ui>
